analyses/batch-effects/util/functions.R

```diff
@@ -0,0 +1,361 @@
+# function used to cut files
+shorten = function(df) {
+  return(df[1:100,])
+}
+
+# function used on kallisto data for summarizing transcripts at the gene level
+grouper = function(df){
```

Comment:
I like to start my custom functions with a verb and make the names as descriptive as possible. When I revisit my code months later, this approach is super helpful for getting me back up to speed!

Something like the following tells me a lot about what the function is doing:

```suggestion
collapse_to_gene_mean = function(df){
```

Reply:
I'll go ahead and make this change